sand/extensions/search/default_search.py

```diff
@@ -0,0 +1,72 @@
+from typing import List
+import re
+from sand.extension_interface.search import IEntitySearch, IOntologySearch
+from sm.misc.funcs import import_attr
+from sand.models.search import SearchResult
+from sand.config import SETTINGS
+
+
+class DefaultSearch(IEntitySearch, IOntologySearch):
+
+    def __init__(self):
+        self.default_classes = import_attr(SETTINGS['ont_classes']['default'])
+        self.default_entities = import_attr(SETTINGS['entity']['default'])
+        self.default_properties = import_attr(SETTINGS['ont_props']['default'])
+
+    def local_search(self, default_entities: dict, search_text: str) -> List:
+        """ performs local partial text search across default entities"""
+        query_tokens = re.findall(r'[a-z]+|\d+', search_text.lower())
```

Comment:
should it be [a-z\d]+? what example did you see contain numbers?

Reply:
as of now we don't have numbers for default entities, if we are never going to have numbers for the default ones, we can remove them. But when I searched through wikidata some of the entities had numbers.
<img width="788" alt="Screenshot 2023-07-11 at 12 22 15 PM" src="https://github.com/usc-isi-i2/sand/assets/32421081/e5e008ef-790a-4149-943d-6030d26d968c">